### PR TITLE
Cb 5277 test components

### PR DESCRIPTION
**Pull request by devnaumov**

### CODE REVIEW

File: webapp/packages/core-blocks/src/Link.test.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        Test Link
+      </Link>,
+      app,
+    );
+
+    const linkElement = await waitFor(() => getByText('Test Link'));
+    fireEvent.click(linkElement);
+
+    expect(handleClick).toHaveBeenCalled();
+  });
```
Comment (sergeyteleshev) [inline webapp/packages/core-blocks/src/Link.test.tsx:62]: please add the case where there is only a `href` and if you click on it - it leads you with this link

File: webapp/packages/core-blocks/src/Link.test.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+  it('should display the indicator icon when indicator is true', async () => {
+    const { container } = renderInApp(
+      <Link href="#" indicator>
+        Test Link
+      </Link>,
+      app,
+    );
+
+    const icon = await waitFor(() => queryByAttribute('href', container, /external-link/i));
```
Comment (sergeyteleshev) [inline webapp/packages/core-blocks/src/Link.test.tsx:33]: searching by `className` here seems a bit easier than by attribute

File: webapp/packages/core-blocks/src/Cell.test.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    expect(beforeText).toBeInTheDocument();
+    expect(afterText).toBeInTheDocument();
+  });
+
+  it('should render description element correctly', async () => {
+    const { getByText } = renderInApp(<Cell description={<span>Description Element</span>}>Test Children</Cell>, app);
+
+    const description = await waitFor(() => getByText('Description Element'));
+    expect(description).toBeInTheDocument();
+  });
```
Comment (sergeyteleshev) [inline webapp/packages/core-blocks/src/Cell.test.tsx:58]: you can also check in new test case that `ripple, big` classnames are present here

File: webapp/packages/core-blocks/src/StatusMessage.test.tsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+import { StatusMessage } from './StatusMessage';
+
+const app = createApp(coreLocalizationManifest, coreDialogsManifest);
+
+describe('StatusMessage', () => {
+  it('should display an error icon and message when type is error', async () => {
+    const message = 'test_error';
+    const { container, getByTitle } = renderInApp(<StatusMessage message={message} type={ENotificationType.Error} />, app);
+    const title = await waitFor(() => getByTitle(message));
+    const icon = await waitFor(() => queryByAttribute('src', container, /error/i));
```
Comment (sergeyteleshev) [inline webapp/packages/core-blocks/src/StatusMessage.test.tsx:24]: why not by className?

Comment (sergeyteleshev) [reply webapp/packages/core-blocks/src/StatusMessage.test.tsx:24]: if possible please change it everywhere

**Review by Wroud — APPROVED** [review]

**Review by sergeyteleshev — APPROVED** [review]

**Pull request merged**